systems/plants/collision/BulletModel.cpp

```diff
@@ -202,6 +204,67 @@ namespace DrakeCollision
     return id;
   }
 
+  bool BulletModel::potentialCollisionPoints(bool use_margins, vector<PointPair> & potential_collisions)
```

Comment:
Why not just create the vector in this function and return it? `potentialCollisions`, which calls this method, now creates the vector the line before `potentialCollisionPoints` is called so it's not like you're saving on allocations. Return value optimization will elide the copy. Returning a boolean that says whether the size is greater than zero is a little superfluous I'd say.